docs/product-user-guide/src/main/asciidoc/drl-rules-central-create-proc.adoc

```diff
@@ -0,0 +1,182 @@
+. After all data objects are in place, return to the *Editor* tab of the DRL designer and define the DRL file with any of the following components:
++
+.Components of a DRL file
+[source]
+----
+package  //automatic
+
+import
+
+global  //optional
+
+function  //optional
+
+query  //optional
+
+declare   //optional
+
+rule
+
+rule
+
+...
+
+----
++
+*<@Michael (2 of 13): I was going to add to this list the keyword `template`, since it's in the list of supported DRL keywords in the Drools doc. I assumed it allowed you to add a rule template to a DRL file, with template keys and such, but I don't think that's right. Right? Again, template keys only apply to rule assets that have an accompanying data table where the values are defined, like guided decision tables and guided rule templates. So template keys are not supported in DRL files, nor `template` keyword, correct? If the `template` keyword is supported, can you tell me how it's used then?>*
++
+* `package`: (automatic) This was defined for you when you created the DRL file and selected the package.
+* `import`: Use this to identify the data objects from either this package or another package that you want to use in the DRL file. Specify the package and data object in the format `package.name.object.name`. You can list multiple data objects on the same `import` line or use multiple lines. *<@Michael (3 of 13): That correct, or do they need to be one per line?>*
++
+.Importing data objects
+[source,java]
+----
+import mortgages.mortgages.LoanApplication;
+----
++
+* `global`: (optional) Use this to include a global variables list to be used by rules in the DRL file. Global variables typically provide data or services for the rules, such as application services used in rule consequences, and to return data from rules, such as logs or values added in rule consequences. Declare the global variable above the rules in the DRL file, use it in an action (`then`) part of the rule, and then set the global value in the decision engine. *<@Michael (4 of 13): Exactly where and how do users set the global value in the working memory? And I've said "decision engine" instead of working memory throughout the doc, because I'd rather not get into defining the working memory, since we say decision engine generally in other docs. Is that still accurate to say?>*
++
+.Declaring and using a global list with a rule
+[source,java]
+----
+global java.util.List myGlobalList;
+
+rule "Using a global"
+  when
+    eval( true )
+  then
+    myGlobalList.add( "My global list" );
+end
+----
++
+.Setting the global list in the decision engine
+[source,java]
+----
+List list = new ArrayList();
+KieSession kieSession = kiebase.newKieSession();
+kieSession.setGlobal( "myGlobalList", list );
+----
++
+*<@Michael (5 of 13): Per question above, where does the user add the above snippet for the decision engine?>*
```

Comment:
@sterobin It's a runtime requirement. KIE Server/Decision Service provides a ```REST``` command to set it. There might be a mechanism to set it in a ```kmodule.xml``` IDK... The code you show above is fine for the standalone/embedded examples in this PR.

Reply:
@sterobin See my other comment about using a "bootstrap" rule to set globals.. @mariofusco I assume that's OK?.. IDK where the check for a global being set is ran. It might be before any rule is matched (e.g. direclty after ```fireAllRules()```) or it might be the first time a rule that tries to use a global triggers.